package.json

```diff
@@ -19,7 +19,7 @@
         "cmdutil": "^1.1.0",
         "cueball": "^2.3.0",
         "extsprintf": "^1.3.0",
-        "fast": "3.1.2",
+        "fast": "3.1.3",
```

Comment:
I should know why this tiny-bump is here, but I don't.  Not a blocker, but I am curious.

Reply:
Discussed on chat, but for posterity:

fast has a dependency on "oldcrc" (an version of crc that produces incorrect values, and we unfortunately need to support both broken and correct crc values). The 3.1.2 version dependency points at a `git://` URL for oldcrc. The 3.1.3 version uses https for that dependency.

joyent/node-fast@e7db8bba (fix)
joyent/node-fast@26f4c67 (node-fast version bump)